modules/tibc/core/04-packet/types/genesis.go

```diff
@@ -43,12 +43,13 @@ func (ps PacketSequence) Validate() error {
 
 // NewGenesisState creates a GenesisState instance.
 func NewGenesisState(
-	acks, receipts, commitments []PacketState,
-	sendSeqs, recvSeqs, ackSeqs []PacketSequence, nextChannelSequence uint64,
+	acks, commitments, receipts []PacketState,
```

Comment:
Why switch the order?

Reply:
To be consistent with the order of the fields in GenesisState

Reply:
If you do it, pls update `gengesis_test.go`.